nsqadmin/logger.go

```diff
@@ -1,5 +1,58 @@
 package nsqadmin
 
+import (
+	"fmt"
+	"strings"
+)
+
 type Logger interface {
 	Output(maxdepth int, s string) error
 }
+
+const (
+	LOG_DEBUG = 1
+	LOG_INFO  = 2
+	LOG_WARN  = 3
+	LOG_ERROR = 4
+	LOG_FATAL = 5
+)
+
+func (n *NSQAdmin) logLevelFromString(level string) int {
+	// check log-level is valid and translate to int
+	switch strings.ToLower(level) {
+	case "debug":
+		return LOG_DEBUG
+	case "info":
+		return LOG_INFO
+	case "warn":
+		return LOG_WARN
+	case "error":
+		return LOG_ERROR
+	case "fatal":
+		return LOG_FATAL
+	case "":
+		return LOG_INFO
```

Comment:
I can change these easily, concern would be if log-level was missing from the .cfg; a blank case would silently load a default rather than fail.

Likewise if a user had `Debug` in a config instead of `debug`, may take them a moment to notice case matters.